.gitignore

```diff
@@ -34,3 +33,106 @@
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+
+# iOS
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
+
+# Thumbnails
+._*
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.com.apple.timemachine.donotpresent
+
+# Directories potentially created on remote AFP share
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+
+# Cookie cutter
+# to ignore any cookie cutter generated files and directories
+
+cookiecutter.json
+{{ cookiecutter.*
+
+# Xcode
+#
+# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
+
+## User settings
+xcuserdata/
+
+## compatibility with Xcode 8 and earlier (ignoring not required starting Xcode 9)
+*.xcscmblueprint
+*.xccheckout
+
+# CocoaPods
+#
+# We recommend against adding the Pods directory to your .gitignore. However
+# you should judge for yourself, the pros and cons are mentioned at:
+# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
+#
+Pods/
+
+# fastlane
+#
+# It is recommended to not store the screenshots in the git repo.
+# Instead, use fastlane to re-generate the screenshots whenever they are needed.
+# For more information about the recommended setup visit:
+# https://docs.fastlane.tools/best-practices/source-control/#source-control
+
+fastlane/README.md
+fastlane/report.xml
+fastlane/Preview.html
+fastlane/screenshots/**/*.png
+fastlane/test_output
+fastlane/xcov_output
+.env
+
+# Sourcery
+*.generated.swift
+
+# Derived Data folder
+DerivedData
+
+# Tuist
+.tuist-generated
+
+# Firebsae
+GoogleService-Info.plist
+
+# Android
+*.iml
+.gradle
+build/
+/captures
+.externalNativeBuild
+signing.properties
+config/release.keystore
+config/release.jks
+config/private_key.pepk
+config/release_keystore_base64_encoded.txt
+
+google-services.json
+
+# KMM
+/local.properties
+.idea
+/build
+*/build
+.cxx
+local.properties
```

Comment:
I don't think we need these git ignore at root level 🤔

Reply:
When create a new KMM project from the Android studio, we will have this .gitignore:

<img width="291" alt="Screen Shot 2022-09-07 at 09 16 46" src="https://user-images.githubusercontent.com/19943832/188773814-003acc21-33e3-4dba-a036-19bdbcf320f4.png">

So I think we can keep it? 🤔